node/actors/bft/src/tests.rs

```diff
@@ -9,7 +9,10 @@ async fn run_test(behavior: Behavior, network: Network) {
 
     const NODES: usize = 11;
     let mut nodes = vec![behavior; NODES];
-    for n in &mut nodes[0..validator::threshold(NODES)] {
+    // validator::threshold(NODES) will calculate required nodes to validate a message
+    // assuming uniform weight distribution
```

Comment:
it would be better to make test accept [(Behavior,weight)] rather than [Behavior] here. This way we won't need to assume stuff about the `Test::run()`.

Reply:
Problem here is that we have to calculate the amount of honest nodes, and for that we are using these assumptions:
- each validator.weight = 1 (which is the default for tests)
- total_weight = number of nodes
I'll change the comment to explain this.

Reply:
Ok, I misunderstood your first comment